server/www/templates/posts/post-editor.html

```diff
@@ -149,8 +149,22 @@ <h1 class="page-title"><span class="icon-left fa-pencil" translate translate-val
                       >
                           {{attribute.label}}
                       </label>
+                      <!-- type: checkbox -->
+                      <div class="form-field checkbox" ng-if="isCheckbox(attribute)" ng-repeat="option in attribute.options">
+                          <input 
+                              type="checkbox" 
+                              checklist-model="post.values[attribute.key]" 
+                              ng-required="attribute.required" 
+                              checklist-value="option" 
+                              value="option"
+                              ng-class="{ 'error': form['values_' + attribute.key].$invalid && form['values_' + attribute.key].$dirty }">
+                         <label>
+                              {{option}} 
+                          </label>
+                      </div>
+
                       <div ng-repeat="(key, value) in post.values[attribute.key] track by key">
-                          <div ng-class="{ 'input-inline' : canRemoveValue(attribute, key) }">
+                          <div ng-if="!isCheckbox(attribute)">
```

Comment:
... or maybe that doesn't matter since it seems we never exposed cardinality to the user